### PR TITLE
Allow searching of servers by IP

### PR DESCRIPTION
**Pull request by aStonedPenguin**
Quality of life change.

### CODE REVIEW

**Comment by robotboy655** [comment]
```
[HTML] TypeError: Cannot call method 'indexOf' of undefined
    at asset://garrysmod/html/js/menu/menuApp.js?2:29:33
    at Array.forEach (native)
    at Object.m [as forEach] (asset://garrysmod/html/js/thirdparty/angular.js:6:192)
    at Object.<anonymous> (asset://garrysmod/html/js/menu/menuApp.js?2:25:11)
    at asset://garrysmod/html/js/thirdparty/angular.js:67:353
    at asset://garrysmod/html/js/thirdparty/angular.js:128:274
    at asset://garrysmod/html/js/thirdparty/angular.js:66:506
    at asset://garrysmod/html/js/thirdparty/angular.js:128:280
    at asset://garrysmod/html/js/thirdparty/angular.js:66:506
    at Object.$eval (asset://garrysmod/html/js/thirdparty/angular.js:87:38)
```

**Comment by robotboy655** [comment]
```
[HTML] TypeError: Cannot call method 'indexOf' of undefined
    at asset://garrysmod/html/js/menu/menuApp.js?2:29:33
    at Array.forEach (native)
    at Object.m [as forEach] (asset://garrysmod/html/js/thirdparty/angular.js:6:192)
    at Object.<anonymous> (asset://garrysmod/html/js/menu/menuApp.js?2:25:11)
    at asset://garrysmod/html/js/thirdparty/angular.js:67:353
    at asset://garrysmod/html/js/thirdparty/angular.js:128:274
    at asset://garrysmod/html/js/thirdparty/angular.js:66:506
    at asset://garrysmod/html/js/thirdparty/angular.js:128:280
    at asset://garrysmod/html/js/thirdparty/angular.js:66:506
    at Object.$eval (asset://garrysmod/html/js/thirdparty/angular.js:87:38)
```

**Comment by robotboy655** [comment]
Ok, let me rephrase, input.address is always undefined.

https://github.com/Facepunch/garrysmod/blob/bc5b70e8e0ead1b32b2b29809f8ce15aa7b0888c/garrysmod/html/js/menu/control.Servers.js#L223

I am assuming this is intended for the first searchbar in the gamemode list, not in the serverlist where you were always able to search by IP.

**Comment by aStonedPenguin** [comment]
I'm just going to pretend I noticed gamemode search was a thing and that this didn't just work because IP search was already a thing.
